lisa/tests/base.py

```diff
@@ -1878,7 +1878,17 @@ def get_rtapp_profile(cls, plat_info, **kwargs):
         def add_buffer(task):
             template_phase = task.phases[0]
             wload = template_phase['wload']
-            task = task.with_props(meta={'from_test': True})
+
```

Comment:
Could you try this instead ?
https://github.com/ARM-software/lisa/pull/1771
```
task.with_props(meta=leaf_precedence({'from_test': True}))
```
This will add some sort of frontier in the tree such that instead of the usual combination order (for 4 levels where the property is set in the tree):
```
(root_val & (this_val  & (intermediate_val & (leaf_val))))
```
we will get:
```
(root_val & ((intermediate_val & (leaf_val)) & this_val))
```

Just in case a problem arises with leaf_precedence, please keep this code around as a backup.